### DISCUSSION TITLE
Potential rename for Crosshair mode

### DISCUSSION THREAD

**Discussion by SmokeyStack** [Mojang/minecraft-editor#42] [Feedback]
Tool mode is self explanatory, it is a mode where you can use tools. However, crosshair mode is a bit vague. You use a crosshair? That's it? A proposed solution is to say normal or creative mode. Perhaps even play/testing mode since you can interact with levers and other blocks.

**Comment by minecraftchloe** [comment]
Naming this was tricky! Calling it "creative" or "normal" aren't options, unfortunately. I do like "Play"! 

However, what's your take on the system right now _not_ having the capacity to undo collateral damage if you're playtesting mechanics in the map -- like, a more sophisticated edit/test system would have? We worry about it being misleading.

**Reply by SmokeyStack** [reply]
Unlike Godot or other game engines, Editor's current implementation of "playtesting" just means single block editing. Which means you can't easily roll back collateral damage, RIP project destroyed due to loose wither. So if normal or creative isn't a name, cross hair should be renamed to single block and a new button for playtesting should be added. As https://github.com/Mojang/minecraft-editor/discussions/64 stated, the only true way to playtest is exporting your project.

**Reply by fraysdev** [reply]
I think preview mode kinda makes sense?

**Reply by dacowan** [reply]
> Naming this was tricky! Calling it "creative" or "normal" aren't options, unfortunately. I do like "Play"!
> 
> However, what's your take on the system right now _not_ having the capacity to undo collateral damage if you're playtesting mechanics in the map -- like, a more sophisticated edit/test system would have? We worry about it being misleading.

I would warn against using `Play` because of the dangers of conflating Crosshair mode with Edit/Test.  Perhaps `Block Mode` or `Traditional` ?

**Reply by LateLag** [reply]
I personally think Crosshair is fine as a name, since it is still an editing mode and properly describes the single-block-editing nature of it. Naming it "play" would be misleading.
